packages/third-party/tailwind-preset/README.md

```diff
@@ -0,0 +1,54 @@
+# Lynx Tailwind Preset
+
+A Tailwind CSS preset specifically designed for Lynx, ensuring that only CSS properties supported by Lynx are available as Tailwind utilities.
+
+> **⚠️ Experimental & Unstable**\
+> This preset is currently in experimental stage and may undergo significant changes. We welcome and encourage contributions from the community to help shape its future development. Your feedback, bug reports, and pull requests are invaluable in making this preset more robust and feature-complete.
+
+## Structure
+
+- `src/lynx.ts`: Main preset configuration that reverse-engineered [Tailwind's core plugins](https://github.com/tailwindlabs/tailwindcss/blob/v3/src/corePlugins.js).
+- `src/plugins/lynx/`: Custom plugins as replacement when per-class customization are needed.
+- `src/__tests__/`: Test files to ensure correct utility generation
+
+## Contributing
+
+### Getting Started
```

Comment:
It would be much easier if this package is in a separate repository.

Reply:
@colinaaa well that's just an excuse that the monorepo DX is not good enough?